OneCommunityGlobal/HighestGoodNetworkApp
# SaiDeepak project summary alignment BMDashboard

# Description
This PR is regarding the alignment of the project summary list which is off in the bmdashboard of Phase 2 and also making the project summary page responsive.

## Related PRS (if any):
This is only a frontend PR and can be tested using the development branch in the backend
…

## Main changes explained:
- Modified the ProjectSummary.jsx file
- Added some styling in BMDashboard.css
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as admin or owner user
5. navigate to /bmdashboard and login again
6. verify that the alignment of the content in the project summaries is in place and is not looking off like before. Please input any suggestions for any particular layout or design changes which would make the list and info look appealing.
7. Also, please verify the responsiveness of the table data by inspecting the page and verify alignment for different screen sizes.

## Screenshots or videos of changes:
Before Pic: 
![Screenshot (103)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114311909/939b2b98-9043-408d-af70-9d84edf71198)
After alignment correction:
![Screenshot (102)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114311909/393c847b-4b50-4e9f-8fe1-fd908a30a713)


## Note:
To test the responsiveness, right click and then click on inspect and then select mobile view and select different screen sizes and check alignment.
